lib/RouterMixin.js

```diff
@@ -203,7 +203,12 @@ var RouterMixin = {
 };
 
 function join(a, b) {
-  return (a + b).replace(/\/\//g, '/');
+  if ((a+b).search(/^http/i) > -1) {
```

Comment:
This is called when the component is created. I'm not sure why // gets replaced with a single / -- but in the case of a full url link we do not want that behavior. Or else our <Link> element's <a href={link}> would be http:/example.com instead of http://example.com